_assignments/mp4.md

```diff
@@ -0,0 +1,206 @@
+
+
+### UI
+
+You must implement a User Interface for your game. Remember to consult the Oculus Best Practices Guide and share ideas with your teammates. 
+If you are still confused, think about the games you have played before, and check [this](https://developer.oculus.com/blog/introducing-the-oculus-sample-framework-for-unity-5/).
+
+1. You should have a pause button that pauses the game, and provides the option to go back to the main menu.
+
+1. Your UI should show the score to players. After hitting a sphere in your scene, your score should increase.
+
+
+### Tutorial
+
+Since you came up with some fancy control scheme, the likes of which we may never have seen before, you must give us a tutorial. 
+You can assume we know how to use the pointing device to access the start menu, but nothing else.
+
+1. Your tutorial must cover all of the plane controls.
+
+1. The tutorial can be accessed from the main menu, or simply be in the same room as the start menu. 
+
+1. For 20 points of extra credit, make your tutorial interactive. This means the player should initiate the tutorial, and receive 
+   prompts to perform actions. Upon performing these actions, the player should receive feedback, and the tutorial should advance. 
+   For example, the tutorial could prompt the player to “Throw the controller across the room in order to accelerate”. [**\***](#footnotes) Upon hurling the controller, 
+   the player would see some indication that the plane has sped up, and get a message saying “good job”.
+
+
+### Sound
+
+Add all necessary sounds to your simulator. Some examples include engine noise, crash sounds, firing sounds, ambient music, etc. Make sure that your 
+sounds are neither too loud, nor too quiet to hear, and make for a comfortable experience.
+
+
+### "Game Over" Condition
+
+When the player crashes the plane, they should see a game over screen, which should give the option to go back to the main menu, restart the game, or quit.
+If your game over screen has a menu, you can choose how it will be controlled, but make sure this is communicated to the player via the tutorial, or messages 
+included in the “game over” menu screen (such as “juggle the touch controllers to return to the main menu”). [**\*\***](#footnotes)
+
+
+### Overall Design
+
+“Overall Design” is a vague definition, but we are going to evaluate it along the following dimensions:
+
+1. Your interface/instruction system works as expected: a first time player, without any prior knowledge of your simulator, should be able to pick up the controls quickly and easily.
+
+1. The overall experience of flying your plane should be smooth and comfortable. For example: Are rotations comfortable? Does your plane behave as expected? Does the weapon work as expected?
+
+1. Your flight simulator should be polished. That is, there should be few, if any, noticeable glitches, controls should work as intended, and the frame rate should be consistently at or above 60fps.
+
+---
+
+#### Footnotes

```

Comment:
Minor issue: Can we change the font size of Footnotes to the same size as other section headings? When the "*" or "**" link to the section, the top bar covers "footnotes" and part of "Don't do this". Other than this, things look good to me.